GIFrameworkMaps.Web/Views/ManagementSearchDefinition/EditLocalSearchDefinition.cshtml

```diff
@@ -0,0 +1,87 @@
+﻿@model GIFrameworkMaps.Data.Models.Search.LocalSearchDefinition
+@{
+    ViewData["Title"] = $"Edit search definition '{Model.Name}'";
+}
+
+<h1>@ViewData["Title"]</h1>
+<div class="row mt-2 mb-2">
+    <div class="col-md-8">
+        <form asp-action="EditLocalSearchDefinition">
+            <div asp-validation-summary="ModelOnly" class="alert alert-danger mb-3"></div>
+            <input type="hidden" asp-for="Id" />
+            <div class="mb-3">
+                <label asp-for="Name" class="control-label"></label>
+                <input asp-for="Name" class="form-control" />
+                <div class="form-text">A friendly name for administrators</div>
+                <span asp-validation-for="Name" class="text-danger"></span>
+            </div>
+
+            <div class="mb-3">
+                <label asp-for="Title" class="control-label"></label>
+                <input asp-for="Title" class="form-control" />
+                <div class="form-text">The name that will appear in the search results list</div>
+                <span asp-validation-for="Title" class="text-danger"></span>
+            </div>
+
+            <div class="mb-3">
+                <label asp-for="AttributionHtml" class="control-label"></label>
+                <input asp-for="AttributionHtml" class="form-control" />
+                <span asp-validation-for="AttributionHtml" class="text-danger"></span>
+            </div>
+
+            <div class="mb-3">
+                <label asp-for="MaxResults" class="control-label"></label>
+                <input asp-for="MaxResults" class="form-control" />
+                <span asp-validation-for="MaxResults" class="text-danger"></span>
+            </div>
+
+            <div class="mb-3">
+                <label asp-for="ZoomLevel" class="control-label"></label>
+                <input asp-for="ZoomLevel" class="form-control" />
+                <span asp-validation-for="ZoomLevel" class="text-danger"></span>
+            </div>
+
+            <div class="mb-3">
+                <label asp-for="EPSG" class="control-label"></label>
+                <select asp-for="EPSG" class="form-select">
+                    <option value="27700">27700 - British National Grid</option>
+                    <option value="4326">4326 - WGS 84 World Geodetic System</option>
+                    <option value="3857">3857 - WGS 84 Pseudo-Mercator</option>
+                </select>
+                <span asp-validation-for="EPSG" class="text-danger"></span>
+            </div>
+
+            <div class="mb-3">
+                <label asp-for="ValidationRegex" class="control-label"></label>
+                <input asp-for="ValidationRegex" class="form-control" />
+                <span asp-validation-for="ValidationRegex" class="text-danger"></span>
+            </div>
+
+            <div class="mb-3">
```

Comment:
We might be able put the parts that are repeated across all search definition types into a partial, so we don't have to repeat ourselves